readthedocs/proxito/views/serve.py

```diff
@@ -151,7 +151,7 @@ class ServeDocs(SettingsOverrideObject):
 
 class ServeError404Base(ServeRedirectMixin, ServeDocsMixin, View):
 
-    def get(self, request, proxito_path, template_name='404.html'):
+    def get(self, request, proxito_path, *args, template_name='404.html', **kwargs):
```

Comment:
I think this is not correct. The function definition is:

```
 defaults.page_not_found(request, exception, template_name='404.html')
```

There is no `*args` or `**kwargs` on it.

What we need to change instead is https://github.com/readthedocs/readthedocs.org/blob/3f6abb500cc5a13e6c7152113aa3b006260e65fa/readthedocs/proxito/tests/handler_404_urls.py#L24-L31 if anything.